cli/azd/test/functional/aspire_test.go

```diff
@@ -240,6 +244,47 @@ func Test_CLI_Aspire_Deploy(t *testing.T) {
 	require.NoError(t, err)
 }
 
+// cleanupDeployments deletes all subscription level deployments tagged with `azd-env-name` equal to envName. If the session
+// indcates we are in playback mode, this function is a no-op.
+func cleanupDeployments(ctx context.Context, t *testing.T, azCLI *azdcli.CLI, session *recording.Session, envName string) {
```

Comment:
My strategy to figure out what tests needed to call this was to look for calls to `NewCLI` where we pass in a `*Session` and arrange for these tests to call `cleanupDeployments`.

Reply:
The engsys cleanup script now also deletes sub-wide deployments. Thanks for looking for these. I couldn't find any others in my cursory searches... but my knowledge of the test suite is not as exhaustive as it was previously.